### PR TITLE
Add namespaces to System.Threading tests

### PR DESCRIPTION
**Pull request by hughbe**
- Part of #2898
- Allows me to remove some usings

- Not sure why the diff has to be so confusing, sorry

### CODE REVIEW

**Comment by JonHanna** [comment]
:+1:

File: src/System.Threading/tests/AsyncLocalTests.cs [hunk 1]
```diff
@@ -2,366 +2,368 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Xunit;
-using System;
-using System.Threading;
 using System.Threading.Tasks;
 
+using Xunit;
```
Comment (stephentoub) [inline src/System.Threading/tests/AsyncLocalTests.cs]: Alphabetizing the namespaces when you're making other changes to the same file is ok, but please don't add the blank line, and please don't modify files if the only thing you're doing is changing the order of the namespaces, etc.  We're really trying to stay away from PRs/commits that only change stylistic things like this, and would prefer to do such changes holistically at some point in an automated fashion.

Comment (hughbe) [inline src/System.Threading/tests/AsyncLocalTests.cs]: Sorry! I reverted this change and rebased.

Comment (stephentoub) [inline src/System.Threading/tests/AsyncLocalTests.cs]: Thanks.  Can you remove the blank lines you added the separate the ```using System...```s from the ```using Xunit;```?

**Comment by stephentoub** [comment]
LGTM. Thanks for fixing these, @hughbe.

**Comment by akoeplinger** [comment]
@hughbe 
> Not sure why the diff has to be so confusing, sorry

FYI if you add [?w=1](https://github.com/dotnet/corefx/pull/6207/files?w=1) to the diff url you'll get a view where whitespace changes are ignored, making the diff much more useful.

**Comment by hughbe** [comment]
Oh awesome, good shout. Thanks

**Pull request merged**